File: CS/DXGridAndGauge/Data/ComboValue.cs
using DXGridAndGauge.Core;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace DXGridAndGauge.Data {
    public class ComboValue : Bindable {
        public int ID {
            get { return id; }
            set {
                if (id == value) return;
                id = value;
                OnPropertyChanged("ID");
            }
        }
        public string Text {
            get { return text; }
            set {
                if (text == value) return;
                text = value;
                OnPropertyChanged("Text");
            }
        }
        int id = -1;
        string text = null;
    }
}

File: VB/DXGridAndGauge/Data/ComboValue.vb
Imports Microsoft.VisualBasic
Imports DXGridAndGauge.Core
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace DXGridAndGauge.Data
	Public Class ComboValue
		Inherits Bindable
		Public Property ID() As Integer
			Get
				Return id_Renamed
			End Get
			Set(ByVal value As Integer)
				If id_Renamed = value Then
					Return
				End If
				id_Renamed = value
				OnPropertyChanged("ID")
			End Set
		End Property
		Public Property Text() As String
			Get
				Return text_Renamed
			End Get
			Set(ByVal value As String)
				If text_Renamed = value Then
					Return
				End If
				text_Renamed = value
				OnPropertyChanged("Text")
			End Set
		End Property
		Private id_Renamed As Integer = -1
		Private text_Renamed As String = Nothing
	End Class
End Namespace
